mage_ai/data_preparation/shared/secrets.py

```diff
@@ -1,10 +1,8 @@
-from cryptography.fernet import Fernet, InvalidToken
-from mage_ai.data_preparation.repo_manager import (
-    get_data_dir,
-    get_repo_path
-)
-from typing import List
 import os
+from typing import List
+
+from cryptography.fernet import Fernet, InvalidToken
+from mage_ai.data_preparation.repo_manager import get_data_dir, get_repo_path
```

Comment:
we organize the imports with the `from` imports first

Reply:
sorry accidentally committed code changes from auto-formatting. Have reverted the formatting